docs/reference/setup/important-settings/discovery-settings.asciidoc

```diff
@@ -55,8 +56,9 @@ cluster.initial_master_nodes: <3>
    - master-node-b
    - master-node-c
 --------------------------------------------------
-<1> The port will default to `transport.profiles.default.port` and fallback to
-    `transport.port` if not specified.
+<1> The port will default to `transport.profiles.default.port` and fall back to
+    `transport.port` if not specified. If neither of these settings is set then
+    the default port is `9300`.
```

Comment:
I would update the formatting/whitespace here as well:

```
<1> The `port` is determined by checking the following settings in order:
+
--

. `transport.profiles.default.port`
. `transport.port`

If none of these settings are specified, the port defaults to `9300`.
--
```

Reply:
Hmm. I think even this is too much for this introductory description; it is rare to need to adjust the default port, and this seems to cause more confusion than it saves. I elected to remove mention of these settings and say the port _usually_ defaults to `9300` with a link to more detailed docs.

Reply:
That works for me. Thanks @DaveCTurner.